src/System.Threading.ThreadPool/tests/ThreadPoolTests.cs

```diff
@@ -30,10 +30,13 @@ public static void ConcurrentInitializeTest()
                 {
                     countdownEvent.Signal();
                     countdownEvent.Wait(ThreadTestHelpers.UnexpectedTimeoutMilliseconds);
-                    Assert.True(ThreadPool.SetMinThreads(processorCount, processorCount));
                 };
 
+            // ThreadPool.SetMinThreads returns false on UapAot
+            ThreadPool.SetMinThreads(processorCount, processorCount);
```

Comment:
The test was asserting that `SetMinThreads` returned true.  I will revert changes to this test and disable it for UapAot.